recipes/exdown/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "exdown" %}
+{% set version = "0.8.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 06a72db1b3481448082a4680a4af8e42f58d2926044705292aa77107939b64c7
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - pytest
```

Comment:
Is pytest really a runtime dependnecy?
```suggestion
```

Reply:
Yes, optional though. (exdown can generate pytest tests from markdown files.) https://github.com/nschloe/exdown/blob/main/exdown/main.py#L79